src/dynarec/rv64/dynarec_rv64_emit_shift.c

```diff
@@ -1429,6 +1435,9 @@ void emit_shld32(dynarec_rv64_t* dyn, int ninst, rex_t rex, int s1, int s2, int
     } else IFX(X_ALL) {
         SET_DFNONE();
     }
+
+    BEQZ_MARK(s5);
```

Comment:
It seems for now `emit_shld32` only gets called once in `SHLD Ed, Gd, CL`, and there is already a `BEQZ_NEXT` on s5 before the function call.